app/templates/motivationPhrasesPageOrel.template.html

```diff
@@ -1,7 +1,8 @@
-<div class="body">
+<div class="wrapper">
   <header class="header_footer header">
     <p class="welcome">Welcome to your todo list</p>
   </header>
+  <div class="body">
```

Comment:
could you please add a more descriptive class name because it is a little bit confusing?

Reply:
thanks for comment. fixed it